libethereum/State.cpp

```diff
@@ -332,10 +332,10 @@ u256 State::balance(Address const& _id) const
 void State::noteSending(Address const& _id)
 {
 	Account* a = account(_id);
-	if (asserts(a != nullptr))
```

Comment:
Did that cause an exception before? Please check where it was caught and increase the logging there.